# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/iproute2/hotfix-limits.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- iproute2-6.4.0/bridge/mdb.c.vanilla	2023-07-09 00:09:20.570343270 +0200
+++ iproute2-6.4.0/bridge/mdb.c	2023-07-09 00:09:43.108717784 +0200
@@ -15,6 +15,7 @@
 #include <string.h>
 #include <arpa/inet.h>
 #include <netdb.h>
+#include <limits.h>
 
 #include "libnetlink.h"
 #include "utils.h"
